lib/utils.py

```diff
@@ -58,3 +60,114 @@ def build_check(target: Path) -> bool:
         Path.unlink('.tmp.config')
 
     return True
+
+def linecount(file_path: Path) -> int:
+    '''Returns the number of lines in a file'''
+    return len(file_path.open().readlines())
+
+def run_cleaned_iwyu (iwyu: List[str], cleaner: List[str],
+                      fix_includes: List[str], debug: bool) -> bool:
+    '''Spawns subprocesses to run the IWYU command,
+    remove the quotes, and transform the original file'''
+    try:
+        _, err = subprocess.Popen(iwyu, stderr=subprocess.PIPE, text=True).communicate()
+        out, _ = subprocess.Popen(cleaner, stdin=subprocess.PIPE,
+                                  stdout=subprocess.PIPE, text=True).communicate(input=err)
+        if debug:
+            print("debug")
+            print(out)
+        change, _ = subprocess.Popen(fix_includes, stdin=subprocess.PIPE,
+                                     stdout=subprocess.PIPE, text=True).communicate(out)
+    except subprocess.CalledProcessError:
+        warn("IWYU FAILED TO RUN")
+        return False
+
+    print(change)
+
+    if "IWYU edited 0 files on your behalf." in change:
+        return False
+    return True
+
+def perform_iwyu(fixer_path: Path, part: json, filters: List[Path],
```

Comment:
is `json` right here? I'm curious if we even need to import json at all in this file?

Reply:
Then can you remove `import json` from this file?